AIProxyFal/AIProxyFal.xcodeproj/project.pbxproj

```diff
@@ -7,16 +7,20 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		2C20F2992CB0E755001ECE32 /* ImageToVideoView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2C20F2982CB0E755001ECE32 /* ImageToVideoView.swift */; };
+		2C20F29C2CB0EB31001ECE32 /* AIProxy in Frameworks */ = {isa = PBXBuildFile; productRef = 2C20F29B2CB0EB31001ECE32 /* AIProxy */; };
+		2C885B432CB1CD0A00C23BAD /* AIProxy in Frameworks */ = {isa = PBXBuildFile; productRef = 2C885B422CB1CD0A00C23BAD /* AIProxy */; };
+		2C885B572CB2B83300C23BAD /* AIProxy in Frameworks */ = {isa = PBXBuildFile; productRef = 2C885B562CB2B83300C23BAD /* AIProxy */; };
```

Comment:
Looks like AIProxy has been added multiple times. You could try removing it and then running File > Packages > Reset Package Caches. Then add AIProxy again